data/categories.txt

```diff
@@ -1426,7 +1426,7 @@ nl:^Computerwinkel
 fi:^Tietokonekauppa
 fr:^Magasin d'informatique
 de:^Computergeschäft
-hu:^Számítógép üzlet
+hu:^Számítógépüzlet
```

Comment:
Are you sure? I see both forms being used.

Reply:
​According to Hungarian grammar, they should be written together.​ (It is so even many people don't know this and write it separated. I think it better for maps.me to be correct.)